content/docs/user-guide/hydra-integration.md

```diff
@@ -0,0 +1,251 @@
+# Hydra Integration
+
+_New in DVC 2.25.0 (see `dvc version`)_
+
+In this guide we explain how to enable and use the DVC and
+[Hydra](https://hydra.cc/) integration.
```

Comment:
What do you think about explaining a little more here? What about something like:

```
In this guide we explain how to configure experiments with Hydra and track them with DVC.
```

Reply:
Went with:

In this guide we explain how to use [Hydra](https://hydra.cc/) composition
for configuring DVC <abbr>experiments</abbr>.

Sounds ok?